packages/application-shell/src/components/navbar/navbar-new/navbar.tsx

```diff
@@ -50,6 +50,8 @@ import compiledStyles from /* preval */ './navbar.styles';
 
 const styles = compiledStyles.jsonMap;
 
+const menuItemHeightInPx = 56;
```

Comment:
**nit**: do you think it would make sense to move this value to the [constants file](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/application-shell/src/constants.ts#L1)?

Reply:
Thank you, that's a great idea 🙌